x/participationrewards/keeper/callbacks.go

```diff
@@ -165,21 +166,25 @@ func OsmosisPoolUpdateCallback(ctx sdk.Context, k *Keeper, response []byte, quer
 }
 
 func OsmosisClPoolUpdateCallback(ctx sdk.Context, k *Keeper, response []byte, query icqtypes.Query) error {
-	var pd concentratedliquidity.ConcentratedPoolExtension
-	if err := k.cdc.UnmarshalInterface(response, &pd); err != nil {
+	var pd model.Pool
+	if err := k.cdc.Unmarshal(response, &pd); err != nil {
 		return err
 	}
```

Comment:
Given that prior unmarshalling problems weren't trivially detected, I highly encourage checking that pd is non-blank/empty after unmarshalling